pkg/network/protocols/http/ebpf_javatls.go

```diff
@@ -98,7 +103,10 @@ func (p *JavaTLSProgram) Start() {
 
 func (p *JavaTLSProgram) Stop() {
 	if p == nil {
+		log.Debugf("java TLS ebpf subprogram is not enabled")
 		return
```

Comment:
you should merge main branch into your branch, and remove any occurrence for `p == nil`
reference https://github.com/DataDog/datadog-agent/pull/14909